src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccountValidationServiceImpl.java

```diff
@@ -2,17 +2,218 @@
 
 import java.util.ArrayList;
 
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.businessobject.ObjectCode;
+import org.kuali.kfs.coa.businessobject.ProjectCode;
+import org.kuali.kfs.coa.businessobject.SubAccount;
+import org.kuali.kfs.coa.businessobject.SubObjectCode;
+import org.kuali.kfs.coa.service.AccountService;
+import org.kuali.kfs.coa.service.ObjectCodeService;
+import org.kuali.kfs.coa.service.ProjectCodeService;
+import org.kuali.kfs.coa.service.SubAccountService;
+import org.kuali.kfs.coa.service.SubObjectCodeService;
+
+import edu.cornell.kfs.concur.ConcurConstants;
 import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
 import edu.cornell.kfs.concur.businessobjects.ValidationResult;
 import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
 
 public class ConcurAccountValidationServiceImpl implements ConcurAccountValidationService {
+    protected AccountService accountService;
+    protected ObjectCodeService objectCodeService;
+    protected SubAccountService subAccountService;
+    protected SubObjectCodeService subObjectCodeService;
+    protected ProjectCodeService projectCodeService;
 
     @Override
     public ValidationResult validateConcurAccountInfo(ConcurAccountInfo concurAccountInfo) {
-        ValidationResult validationResult = new ValidationResult(false, new ArrayList<String>());
+        ValidationResult validationResult = checkAccountingString(
```

Comment:
I would consider eliminating this intermediate variable, validationResult, and just returning the result of the checkAccountingString() call directly.